File: src/main/java/com/procurement/evaluation/EvaluationApplication.java
package com.procurement.evaluation;

import com.procurement.evaluation.config.ApplicationConfig;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.netflix.eureka.EnableEurekaClient;

@SpringBootApplication(scanBasePackageClasses = ApplicationConfig.class)
@EnableEurekaClient
public class EvaluationApplication {

    public static void main(final String[] args) {
        SpringApplication.run(EvaluationApplication.class, args);
    }
}

File: src/main/kotlin/com/procurement/evaluation/EvaluationApplication.kt
package com.procurement.evaluation

import com.procurement.evaluation.infrastructure.configuration.ApplicationConfiguration
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackageClasses = [ApplicationConfiguration::class])
class EvaluationApplication

fun main(args: Array<String>) {
    runApplication<EvaluationApplication>(*args)
}
